it/repositories/CacheRepositorySpec.scala

```diff
@@ -86,7 +86,7 @@ class CacheRepositorySpec extends CacheRepositorySpecBase {
 
       findOne(userAnswers3.lrn, userAnswers3.eoriNumber) should not be defined
 
-      val setResult = repository.set(userAnswers3.metadata, None).futureValue
+      val setResult = repository.set(userAnswers3.metadata, None, None).futureValue
```

Comment:
Can we have a test to ensure if the departureId is populated that it's there in the getResult?